src/pudl/metadata/resources/eia930.py

```diff
@@ -192,6 +192,33 @@
         "etl_group": "eia930",
         "create_database_schema": False,
     },
+    "out_eia930__hourly_aggregated_demand": {
+        "description": (
+            """EXPERIMENTAL / WORK-IN-PROGRESS, 2025-04-04.
+
+This aggregates imputed demand from the ``out_eia930__hourly_operations`` table.
+It aggregates demand to the level of EIA regions, interconnects, and the contiguous
+United States. The spatial granularity of each record is indicated by
+``aggregation_level``."""
+        ),
+        "schema": {
+            "fields": [
+                "datetime_utc",
+                "aggregation_level",
+                "aggregation_group",
+                "demand_imputed_pudl_mwh",
+            ],
+            "primary_key": [
+                "datetime_utc",
+                "aggregation_level",
```

Comment:
Does `aggregation_level` really need to be part of the primary key?

Reply:
If the uniqueness of `aggregation_group` values across different `aggregation_level`'s is incidental then it probably makes sense to keep it in the PK.